cli/src/cluster_query.rs

```diff
@@ -309,12 +309,13 @@ pub fn process_ping(
     let mut submit_count = 0;
     let mut confirmed_count = 0;
     let mut confirmation_time: VecDeque<u64> = VecDeque::with_capacity(1024);
+    let lamport = 0;
```

Comment:
these are needed for testing only. I'll create separate PR for this function addition to `solana ping`.

Reply:
Btw, this is done #7029